test/std-format-test.cc

```diff
@@ -110,7 +110,7 @@ template <> struct std::formatter<S> {
     char c = get_char();
     if (!isdigit(c) || (++iter, get_char()) != '}')
       throw format_error("invalid format");
-    width_arg_id = c - '0';
+    width_arg_id = static_cast<size_t>(c - '0');
```

Comment:
This tests an example from the standard and should remain unchanged.

Reply:
Should width_arg_id become signed then?

Reply:
width_arg_id is passed on the next line to check_arg_id which requires a size_t .